pkg/crd/known_types.go

```diff
@@ -103,6 +105,13 @@ var KnownPackages = map[string]PackageOverride{
 		}
 		p.AddPackage(pkg) // get the rest of the types
 	},
+	"k8s.io/api/core/v1": func(p *Parser, pkg *loader.Package) {
+		p.Schemata[TypeIdent{Name: "Protocol", Package: pkg}] = apiext.JSONSchemaProps{
+			Type:    "string",
+			Default: &apiext.JSON{Raw: []byte(strconv.Quote("TCP"))},
```

Comment:
this isn't quite right... the default isn't inherent to the type, it's contextual (Protocol inside ServicePort and ContainerPort defaults to "TCP", but other uses might not)

Reply:
what do you suggest we do here then?